### ISSUE TITLE
squoosh-cli exits in between batch file

### ISSUE THREAD

**Issue by SteiveSteive** [GoogleChromeLabs/squoosh#1143]
Hi!
I do not know if its a bug report or a feature request so just posting here.

node -v
v14.17.5

npm -v
6.14.14

npm ls -g | find "squoosh"
+-- @squoosh/cli@0.7.2
| +-- @squoosh/lib@0.4.0

OS : Win 10 Pro Ver. 10.0.19043.1165

Normal usage when typing inside the cmd.exe command prompt works fine as follows:

`squoosh-cli --mozjpeg {quality:90} image.jpg
`
What does not work is when the above commands are put inside a batch file.

```
@ECHO OFF
:: FILENAME: batch.cmd
squoosh-cli --mozjpeg {quality:90} image.jpg
ECHO HELLO
```

Which results in just doing and finishing the squoosh-cli only and not going any further like ECHO HELLO.

```
c:\temp\batch.cmd
1/1 √ Squoosh results:
 image.jpg: 227.06KB
  └ image.jpg → 55.94KB (24.6%)
```

Personally, I would like it to be as follows:

```
c:\temp\batch.cmd
1/1 √ Squoosh results:
 image.jpg: 227.06KB
  └ image.jpg → 55.94KB (24.6%)
HELLO
```

Both 
npx @squoosh/cli --mozjpeg {quality:90} image.jpg
and
squoosh-cli --mozjpeg {quality:90} image.jpg
exits in the middle of the batch file.

How can we use squoosh-cli inside a batch file?

**Comment by surma** [comment]
> Which results in just doing and finishing the squoosh-cli only and not going any further like ECHO HELLO.

My _hunch_ is that it is actually printing the “HELLO” but it gets lost through terminal movement commands. Can you re-try your experiment but instead of printing to the console you create a file or something as a marker that the command was executed?

**Comment by SteiveSteive** [comment]
Hi Surma,

Thank you for the reply.

Actually it was created as batch.cmd and executed but the HELLO did not print out on the screen.

I actually got it to work as below after I posted here with some addition.

I modified it afterwards and came out as the image below. (batch name is a6b4.cmd)
![command-prompt-image](https://user-images.githubusercontent.com/90020778/132081786-efa3945c-444a-42ed-b8bb-7a86b91325be.png)

And the batch script
![code](https://user-images.githubusercontent.com/90020778/132081791-24ab402a-eaa7-4214-8201-34dcddbbd8ec.png)

It's been few days since I was thinking of this matter and I first tried to modify the squoosh-cli.cmd inside c:\users\(username)\AppData\Roaming\npm\ which had the "EXIT /b" but nothing changed. Anyway, changing the original codes are not the best things to do so I posted here.

After a while, if there's an EXIT, there should be a START, which was half successful, and then I tried CMD which was very successful for what I was wanting to do inside my script.

So the outcome was to add CMD /C before the squoosh-cli which did the trick.

`cmd /c squoosh-cli --oxipng {level:2,interlace:false} image.png`

Thank you!
